src/preprocessor.ts

```diff
@@ -48,15 +46,12 @@ export function process(
             }
         );
 
-        const outputText =
-            compilerOptions.allowSyntheticDefaultImports && !tsJestConfig.skipBabel
-                ? babelJest.process(
+        const outputText = postHook(
                     tsTranspiled.outputText,
                     path + '.js', // babel-jest only likes .js files ¯\_(ツ)_/¯
```

Comment:
Should we take the opportunity to change the extension based on the source file?  If it's `.tsx`, set it to `.jsx` otherwise set it to `.js`?